### DISCUSSION TITLE
Dynamic Command Loader Evennia

### DISCUSSION THREAD

**Discussion by TonyDeRosia** [evennia/evennia#3698] [Community Contribs & Snippets]
https://github.com/TonyDeRosia/evennia_Dynamic_environment

auto load command files from the dynamic folder.

**Comment by Griatch** [comment]
That's a cool feature! I looked it over and it's really pretty elegant to have it triggered from the cmdset class itself! I wonder if something like this may be interesting for Evennia core as well.

(One note, since you are offering a full game dir - your repo contains both an evennia.db3 file as well as the `secret_settings.py` file with intact secret hash. These should not be part of the repo since they are generated by setup and is different from dev to dev. :slightly_smiling_face: )

**Reply by TonyDeRosia** [reply]
I would love to see it become a core feature. I wanted to say you should check it out: https://github.com/TonyDeRosia/evennia_Dynamic_environment/blob/main/README.md because I was up all night working on it and I think I may have a solid system for it now.
